bucket/sqlcl.json

```
@@ -0,0 +1,23 @@
+{
+    "version": "22.2.0.172.1718",
+    "description": "Oracle SQL Developer Command Line (SQLcl)",
+    "homepage": "https://www.oracle.com/sqlcl",
```

```suggestion
    "homepage": "https://www.oracle.com/database/sqldeveloper/technologies/sqlcl/",
```

It's better to use non-redirecting URLs